mixer/pkg/config/crd/store.go

```diff
@@ -146,12 +150,26 @@ func (s *Store) Init(kinds []string) error {
 	s.informers = make(map[string]cache.SharedInformer, len(kinds))
 	remaining := s.checkAndCreateCaches(d, lwBuilder, kinds)
 	if len(remaining) > 0 {
+		for _, k := range remaining {
+			if s.isCriticalKind(k) {
+				panic(fmt.Errorf("critical kind %s is not ready", k))
+			}
+		}
 		log.Warnf("Failed to discover kinds: %v", remaining)
 	}
```

Comment:
If we fail to discover these `kinds` then we should do
`go func(remaining int) {
   for remaining > 0 { 
           sleep(sleeptime)
           remaining = s.ckeckAndCreateCaches()
   }
}(remaining)

Reply:
Added retry. Done.